docs/api/products.rst

```diff
@@ -425,6 +428,256 @@ Example Response
++=====================================+================+===================================================================================+
+| channels                            | request        | Collection of channels codes, which we want to associate with created product     |
++-------------------------------------+----------------+-----------------------------------------------------------------------------------+
+| translations['locale_code']['name'] | request        | Collection of translations (each contains slug and name in given locale).         |
+| translations['locale_code']['slug'] |                | Only the translation for default locale is required, the rest are optional        |
++-------------------------------------+----------------+-----------------------------------------------------------------------------------+
+| options                             | request        | Collection of options codes, which we want to associate with created product      |
++-------------------------------------+----------------+-----------------------------------------------------------------------------------+
+| images                              | request        | Collection of images codes, which we want to associate with created product       |
++-------------------------------------+----------------+-----------------------------------------------------------------------------------+
+| attributes                          | request        | Array of attributes (each object has information about selected attribute's code, |
+|                                     |                | its value and locale in which it was defined)                                     |
++-------------------------------------+----------------+-----------------------------------------------------------------------------------+
+| associations                        | request        | Object with code of productAssociationType and string in which the codes of       |
+|                                     |                | associated products was written down.                                             |
++-------------------------------------+----------------+-----------------------------------------------------------------------------------+
+| product_taxons                      | request        | String in which the codes of taxons was written down (separated by comma)         |
++-------------------------------------+----------------+-----------------------------------------------------------------------------------+
+| main_taxon                          | request        | The main taxon's code to whose product is assigned                                |
++-------------------------------------+----------------+-----------------------------------------------------------------------------------+
+
+Example
+.......
```

Comment:
After @TheMadeleine reviewed my docs, it appears that we have to change a few of headlines 
```
Example
^^^^^^^
```